pwiz_tools/Skyline/app.config

```diff
@@ -690,7 +690,7 @@
             <setting name="MixedMoleculeSettingsTab" serializeAs="String">
                 <value>0</value>
             </setting>
-            <setting name="Prosit" serializeAs="String">
+            <setting name="Koina" serializeAs="String">
```

Comment:
Leave these changes for Brian Pratt's static analysis PR. Stick to only the one settings change required for this PR.